sql/changes/1.13/add-user-login.sql

```diff
@@ -0,0 +1,45 @@
+
+create table session_history (
+  session_id int primary key,
+  users_id   int not null references users(id),
+  created    timestamp without time zone not null default current_timestamp,
+  last_used  timestamp without time zone,
+  ended      timestamp without time zone,
+  termination text -- 'logout' / 'expired'
```

Comment:
The column name should really be 'termination_reason', 'termination_type' or 'ending_reason'

Reply:
Renamed to 'termination_reason'.